spec/requests/apartments_spec.rb

```diff
@@ -189,7 +189,486 @@
     expect(response).to have_http_status(422)
     json = JSON.parse(response.body)
     expect(json['user_id']).to include "can't be blank"
-end
+  end
+  describe "PATCH /update" do
+    it "updates an apartment" do 
+      apartment_params = {
+        apartment: {
+          bedrooms: 3, 
+          bathrooms: 7, 
+          address: "14 Glamuleon Drive", 
+          planet: "GlipGlop", 
+          square_footage: 2000, 
+          price: 500000, 
+          utilities: "solar panels, slime chamber, washing machine", 
+          pets: true, 
+          parking: "ship hangar", 
+          image: "https://live.staticflickr.com/65535/50077136637_f3611de27c_b.jpg",
+          user_id: user.id
+        }
+      }
+      post "/apartments", params: apartment_params
+      apartment = Apartment.first
+      updated_apartment_params = {
+        apartment: {
+          bedrooms: 3, 
+          bathrooms: 7, 
+          address: "14 Glamuleon Drive", 
+          planet: "GlipGlop", 
+          square_footage: 2000, 
+          price: 500000, 
+          utilities: "solar panels, slime chamber, washing machine", 
+          pets: true, 
+          parking: "ship hangar", 
+          image: "TESThttps://live.staticflickr.com/65535/50077136637_f3611de27c_b.jpg",
```

Comment:
Just a thought, you can make the update something that will stand out a bit more.